### PR TITLE
OAK-8950 DataStore: FileCache should use one cache segment

### PR DESCRIPTION
**Pull request by thomasmueller**
PR for review (please don't merge, just review)

### CODE REVIEW

File: oak-blob-plugins/src/test/java/org/apache/jackrabbit/oak/plugins/blob/FileCacheTest.java [hunk 1]
```diff
@@ -317,15 +317,17 @@ public void evictExplicit() throws Exception {
     public void evictImplicit() throws Exception {
         LOG.info("Started evictImplicit");
 
+        cache = FileCache.build(60 * 1024/* KB */, root, loader, null);
+
```
Comment (amit-jain) [inline oak-blob-plugins/src/test/java/org/apache/jackrabbit/oak/plugins/blob/FileCacheTest.java:325]: I think good to add `closer.register(cache);` so, all threads are closed after test.

**Comment by amit-jain** [comment]
@thomasmueller Apart from a small addition in the test PR lgtm.

File: oak-blob-plugins/src/test/java/org/apache/jackrabbit/oak/plugins/blob/FileCacheTest.java (hunk 1 above)
Comment (thomasmueller) [reply oak-blob-plugins/src/test/java/org/apache/jackrabbit/oak/plugins/blob/FileCacheTest.java:325]: Good finding... Added!

**Comment by thomasmueller** [comment]
Closing as it's merged in svn.

**Pull request closed without merging**